docs/devel/README.md

```diff
@@ -32,22 +32,31 @@ $ go version
 ```
 This documentation is written using Go version 1.11+.
 
-## Task 4. Install or upgrade Docker
+## Task 4. Install or upgrade Docker or Podman
 If you haven't already, install the 
 [Docker software using the instructions for your operating system][docker].
+or
+[Podman software using the instructions for your operating system][podman].
 If you have an existing installation, check your version and make sure you have
-the latest Docker.
+the latest Docker or Podman
 
 To check if `docker` is has been installed:
 ```
 $ docker --version
 ```
 This documentation is written using Docker version 18.09.0.
 
+To check if `podman` is has been installed:
+```
+$ podman --version
+```
+This documentation is written using Podman version 0.12.1.
+
 
 
 [github]: https://github.com/
 [community]: https://github.com/kubernetes/community
 [contributor]: https://github.com/kubernetes/community/blob/master/contributors/guide/README.md
 [golang]: https://golang.org/doc/install
 [docker]: https://docs.docker.com/install/#supported-platforms
+[podman]: https://github.com/containers/libpod/install
```

Comment:
This links doesn't exist.  May be https://github.com/containers/libpod/blob/master/install.md ?

Reply:
Oops

Reply:
FIxed